### PR TITLE
Add register_command and register_steam functions

### PR DESCRIPTION
**Pull request by sairam4123**
Adds [`register_command`](https://discord.com/developers/docs/game-sdk/activities#registercommand) and [`register_steam`](https://discord.com/developers/docs/game-sdk/activities#registersteam) functions.

### CODE REVIEW

File: godotcord_activity_manager.cpp [hunk 1]
```diff
@@ -1,135 +1,147 @@
...
-}
-
-GodotcordActivityManager::GodotcordActivityManager() {
-	ERR_FAIL_COND_MSG(singleton != NULL, "Only one instance of GodotcordActivityManager should exist")
-	singleton = this;
-}
-
-GodotcordActivityManager::~GodotcordActivityManager() {
-
-}
```
Comment (sairam4123) [inline godotcord_activity_manager.cpp:135]: Forgot to add a single newline. 😆

**Comment by sairam4123** [comment]
Ready for merge, I've squashed all the commits into one single commit, so it should be easy for you to rebase.

**Review by summertimejordi — CHANGES_REQUESTED** [review]
I'm having trouble figuring out what exactly these functions do. Also why are we registering Steam IDs?

**Comment by summertimejordi** [comment]
[Ah I see](https://discord.com/developers/docs/game-sdk/activities#registercommand)
The documentation you PR'd should be updated to be more specific. Its pretty confusing.

**Review by summertimejordi — APPROVED** [review]

**Comment by sairam4123** [comment]
Let me link those functions, in the head comment.

**Pull request merged**